specification/purview/resource-manager/Microsoft.Purview/preview/2024-04-01-preview/purview.json

```diff
@@ -1663,6 +1720,11 @@
           "type": "string",
           "readOnly": true
         },
+        "defaultDomain": {
+          "description": "Gets or sets the default domain in the account.",
+          "type": "string",
+          "readOnly": true
```

Comment:
How can it be used to set it, if its readonly?

Reply:
Updated the description to only state Gets the default domain. Also added into the Accounts_Get example.

Reply:
BTW in case it leads to a doc clarification - is this a DNS domain? FQDN? Or something else?

Reply:
It is something else. Domain in this case is an internal term for a Root layer Collection of Objects (in Purview)